LeagueInformer/LeagueInformer/Interfaces/IGetSummonerGame.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using LeagueInformer.Models;
+
+namespace LeagueInformer.Interfaces
+{
+    public interface IGetSummonerGame
+    {
+        /// <summary>
+        /// Gets information about summoner with given nickname
```

Comment:
XML-Doc jest błędny - "Gets informations about game where is summoner with given nickname" czy coś w ten deseń